addon/config/ur.js

```diff
@@ -1,2 +1,5 @@
 import en from './en';
-export default en;
+
+export default {
+  rtl: true
+};
```

Comment:
You'll need some pluralization rules. If you want the ones from English (1 = "one", everything else is "other"), you could do
```js
import en from './en'
export default {
  pluralForm: en.pluralForm,
  rtl: true
}
```